source/common/http/utility.cc

```diff
@@ -329,5 +330,43 @@ const std::string& Utility::getProtocolString(const Protocol protocol) {
   NOT_REACHED;
 }
 
+void Utility::extractHostPathFromUri(const std::string& uri, std::string& host, std::string& path) {
```

Comment:
since the method just does substr `uri`, can the output be `absl::string_view`?

Reply:
Done